src/services/io/podio/make_datamodel_glue.py

```diff
@@ -73,13 +73,18 @@ def AddCollections(datamodelName, collectionfiles):
         header = '#include <'+ datamodelName + header_fname + '>'
         header_lines.append(header)
 
+        type_map.append('namespace ' + datamodelName + ' {')
```

Comment:
```suggestion
        type_map.append('namespace ' + datamodelName.lower() + ' {')
```
Is this needed? I thought the entry point headers for the data models are EDM4hep.h etc right? On my phone, hard to check.

Reply:
It's already lowecase, see:
https://github.com/eic/EICrecon/pull/642/files#diff-a4f8457644f0e66b97c328c3b74947e07aa878793c0bbed9cdf84a596188a597L91-L92